### PR TITLE
Latest updates

### PR DESCRIPTION
**Pull request by ArthurZaharov**
- Change "domain" to "site"
- Change "uncategorized" to "miscellaneous"
- Add security headers email template
- Add "My sites" tab

### CODE REVIEW

**Comment by codelion** [comment]
This fixes https://github.com/srcclr/security-headers/issues/17

File: assets/javascripts/discourse/templates/headlines/favorite-domains.hbs [hunk 1]
```diff
@@ -35,4 +33,8 @@
       {{/each}}
     </table>
   {{/if}}
-</div>
+{{else}}
+  <div class="text--center">
+    Please <a href="#" class="color--primary" {{action "login"}}>log in</a> to build the list of your favorite sites
```
Comment (codelion) [inline assets/javascripts/discourse/templates/headlines/favorite-domains.hbs]: Can we please change it to "Please log in to add your sites and receive scan reports over email"

**Comment by codelion** [comment]
GTM, after fixing the comment I left!

**Comment by codelion** [comment]
@ArthurZaharov can you please make the change in the text I requested and then we can merge this PR.

**Comment by ArthurZaharov** [comment]
Hi @codelion. Done, please check again. Thanks.

**Comment by codelion** [comment]
GTM!

**Comment by codelion** [comment]
@ArthurZaharov  it failed to build -

```
$ bundle exec rake multisite:migrate
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

undefined method `update' for nil:NilClass/var/www/discourse/plugins/security-headers/db/migrate/20160122082510_rename_uncategorized.rb:6:in `up'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:605:in `exec_migration'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:586:in `block (2 levels) in migrate'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:585:in `block in migrate'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:294:in `with_connection'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:584:in `migrate'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:759:in `migrate'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:998:in `block in execute_migration_in_transaction'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:1044:in `block in ddl_transaction'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/connection_adapters/abstract/database_statements.rb:201:in `block in transaction'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/connection_adapters/abstract/database_statements.rb:209:in `within_new_transaction'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/connection_adapters/abstract/database_statements.rb:201:in `transaction'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/transactions.rb:208:in `transaction'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:1044:in `ddl_transaction'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:997:in `execute_migration_in_transaction'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:959:in `block in migrate'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:955:in `each'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:955:in `migrate'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:814:in `up'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/migration.rb:792:in `migrate'
/var/www/discourse/vendor/bundle/ruby/2.0.0/gems/activerecord-4.1.14.1/lib/active_record/railties/databases.rake:34:in `block (2 levels) in '
/var/www/discourse/vendor/gems/rails_multisite/lib/tasks/db.rake:8:in `block (2 levels) in '
/var/www/discourse/vendor/gems/rails_multisite/lib/rails_multisite/connection_management.rb:86:in `block in each_connection'
/var/www/discourse/vendor/gems/rails_multisite/lib/rails_multisite/connection_management.rb:84:in `each'
/var/www/discourse/vendor/gems/rails_multisite/lib/rails_multisite/connection_management.rb:84:in `each_connection'
/var/www/discourse/vendor/gems/rails_multisite/lib/tasks/db.rake:3:in `block in '
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
Migrating default
---------------------------------
== 20160122082510 RenameUncategorized: migrating ==============================
```

**Comment by ramil350** [comment]
Fixed in: https://github.com/srcclr/security-headers/pull/26

**Pull request merged**